engine/class_modules/sc_demon_hunter.cpp

```diff
@@ -4717,6 +4721,14 @@ struct fracture_t : public demon_hunter_attack_t
       mh->set_target( s->target );
       oh->set_target( s->target );
       mh->execute();
+      // t30 4pc proc happens on main hand execute but before off hand execute
+      // this matters because the off hand hit benefits from Fiery Demise that may be
+      // applied because of the t30 4pc proc
+      if ( p()->buff.t30_vengeance_4pc->up() )
+      {
+        p()->active.fiery_brand_t30->execute_on_target( s->target );
+        p()->buff.t30_vengeance_4pc->reset();
```

Comment:
->reset() is for end of iteration and resets some other things like last_start, last_trigger, etc. Should just use ->expire() here.